src/hooks/useTabNavigatorFocus/index.ts

```diff
@@ -54,12 +54,10 @@ function useTabNavigatorFocus({tabIndex}: UseTabNavigatorFocusParams): boolean {
         if (!tabPositionAnimation) {
             return;
         }
-        const index = Number(tabIndex);
-
```

Comment:
These are just cleanup changes, since we don't need to do this cast here. Not related to the issue